lib/device_impl.toit

```diff
@@ -42,20 +46,23 @@ class FlashStore_ implements device.Store:
   constructor.init_:
 
   get key/string -> any:
-    throw "NOT IMPLEMENTED"
+    bytes := kv_store_.bytes key
+    if not bytes: return null
+
+    return ubjson.decode bytes
 
   delete key/string:
-    throw "NOT IMPLEMENTED"
+    return kv_store_.delete key
```

Comment:
I'd just drop the return. Doesn't look like delete is supposed to return anything (-> none).